csharp_dotnetcore/Using-Cards/Using_Cards/Using_Cards.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.5" />
+    <PackageReference Include="Microsoft.Bot.Builder" Version="4.0.0.38511" />
+    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.0.0.38511" />
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.0.0.38511" />
+  </ItemGroup>
```

Comment:
Missing this StyleCop references and Ruleset file. Please see instructions (via email) how to add, or check out @daveta sample here: 
Ruleset file: https://github.com/Microsoft/BotBuilder-Samples/pull/302/files#diff-5b203eefa356f6d425fdddb93c15745e

Relevant Packages to add:
```
<ItemGroup>
    <PackageReference Include="AsyncUsageAnalyzers" Version="1.0.0-alpha003" PrivateAssets="all" />
    <PackageReference Include="StyleCop.Analyzers" Version="1.1.0-beta008" PrivateAssets="all" />
  </ItemGroup>
```